### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by giswqs**
Fixed several topos using [codespell](https://github.com/codespell-project/codespell).

### CODE REVIEW

File: examples/LegendControl.ipynb [hunk 1]
```diff
@@ -155,7 +155,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Positionning"
+    "### positioning"
```
Comment (davidbrochart) [inline examples/LegendControl.ipynb:158]:
```suggestion
    "### Positioning"
```

File: ipyleaflet/leaflet.py [hunk 2]
```diff
@@ -1841,11 +1841,11 @@ def legends(self, legends):
         self.legend = legends
 
     @property
-    def positionning(self):
+    def positioning(self):
```
Comment (davidbrochart) [inline ipyleaflet/leaflet.py:1844]: Maybe we could keep the old `positionning` and make it call the new `positioning`, for compatibility reasons?

Comment (giswqs) [reply ipyleaflet/leaflet.py:1844]: Just added `positionning`.

**Comment by davidbrochart** [comment]
Thanks!

**Pull request merged**